pages/help/help_domain-management.md

```diff
@@ -15,71 +15,113 @@ Manage DNS information, domain contacts, and more. Please keep information about
 If you don’t already have a Login.gov account, you’ll need to create one to change any information about your .gov domain. [Create a Login.gov account](https://login.gov/help/get-started/create-your-account/){.usa-link--external}.
 
  
-- [Manage DNS information for your domain](#manage-dns-information-for-your-domain)
+- [Manage DNS name servers](#manage-dns-name-servers)
+- [Enable DNSSEC on your domain](#enable-dnssec-on-your-domain)
+- [Disable DNSSEC on your domain](#disable-dnssec-on-your-domain)
 - [Update your organization name or mailing address](#update-your-organization-name-or-mailing-address)
 - [Update information about your authorizing official](#update-information-about-your-authorizing-official)
 - [Update your contact information (email, phone)](#update-your-contact-information-(email%2C-phone))
 - [Add or update the security email for your .gov domain](#add-or-update-the-security-email-for-your-.gov-domain)
-- [Add or update contacts to help manage your domain](#add-or-update-contacts-to-help-manage-your-domain)
+- [Add a domain manager to your .gov domain](#add-a-domain-manager-to-your-.gov-domain)
 
 
-## Manage DNS information for your domain
+## Manage DNS name servers
 
-A name server is a server within the Domain Name System (DNS) that answers queries about a domain name. DNS is the internet service that translates your domain name into an IP address. This makes it possible for people to access your online service by using a name instead of a numbered address. Before your approved .gov domain can be used, you’ll need to connect it to your DNS hosting service. You’ll also need to enter your name server information on the .gov domain management site. Follow the steps below to do that.
+The Domain Name System (DNS) is the internet service that translates your domain name into an IP address. This makes it possible for people to access your online service by using a name instead of a numbered address.
 
-1. [Sign in with your Login.gov account](#)
+Before your approved .gov domain can be used, [you’ll first need to connect it to a DNS hosting service](https://beta.get.gov/domains/moving/#find-dns-hosting-services). After you’ve set up hosting, you’ll need to enter your name server information on the .gov registrar. Name server records indicate which DNS server is authoritative for your domain. You must add at least two name servers (13 max).
+
+1. Sign in to the [.gov registrar](https://manage.get.gov){.usa-link--external}. using your Login.gov account.
```

Comment:
```suggestion
1. Sign in to the [.gov registrar](https://manage.get.gov). using your Login.gov account.
```
No need to use that here, maybe a typo? This will render as plain-text.